src/components/Sponsors/Sponsors.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from '../Sponsors/Sponsors.module.css';
+import Cards from '../../assets/Card.json';
+
+const Sponsors = () => {
+  return (
+    <section className={styles.main_container}>
+      <img
+        className={styles.bg_container}
+        src="images/bg_landing_stars.svg"
+        alt="Background stars"
+      />
+      <img className={styles.text_container} src="images/sponsor_logo.png" />
```

Comment:
Why is this an image??? The font is already imported in App.css

Reply:
Rupantar used it as image. Changed it to text